Mathlib/Probability/Moments/Covariance.lean

```diff
@@ -170,4 +176,142 @@ lemma covariance_const_sub_right [IsProbabilityMeasure μ] (hY : Integrable Y μ
     cov[X, fun ω ↦ c - Y ω; μ] = - cov[X, Y; μ] := by
   simp [sub_eq_add_neg, hY.neg']
 
+lemma variance_sub [IsFiniteMeasure μ] (hX : MemLp X 2 μ) (hY : MemLp Y 2 μ) :
+    Var[X - Y; μ] = Var[X; μ] - 2 * cov[X, Y; μ] + Var[Y; μ] := by
+  rw [← covariance_self, covariance_sub_left hX hY (hX.sub hY), covariance_sub_right hX hX hY,
+    covariance_sub_right hY hX hY, covariance_self, covariance_self, covariance_comm]
+  · ring
+  · exact hY.aemeasurable
+  · exact hX.aemeasurable
+  · exact hX.aemeasurable.sub hY.aemeasurable
+
+lemma variance_fun_sub [IsFiniteMeasure μ] (hX : MemLp X 2 μ) (hY : MemLp Y 2 μ) :
+    Var[fun ω ↦ X ω - Y ω; μ] = Var[fun ω ↦ X ω; μ] -
+      2 * cov[fun ω ↦ X ω, fun ω ↦ Y ω; μ] + Var[fun ω ↦ Y ω; μ] :=
```

Comment:
```suggestion
    Var[fun ω ↦ X ω - Y ω; μ] = Var[X; μ] - 2 * cov[X, Y; μ] + Var[Y; μ] :=
```
I don't think it's worth spelling out the fun when it's a single function, and not an operation.

Same thing below.